### PR TITLE
RDoc-2498: modify id input into a document string id.

### PR DESCRIPTION
**Pull request by LielNagar**

### CODE REVIEW

File: src/unit1/lesson7/README.md [hunk 1]
```diff
@@ -287,10 +287,11 @@ private void RetrieveContact()
 {
     Console.WriteLine("Enter the contact id: ");
     var id = Console.ReadLine();
-
+	var contactsReference = $"contacts/{id}-A";
```
Comment (garayx) [inline src/unit1/lesson7/README.md]: please format the code

File: src/unit1/lesson7/README.md [hunk 2]
```diff
@@ -343,10 +345,11 @@ private void DeleteContact()
 {
     Console.WriteLine("Enter the contact id: ");
     var id = Console.ReadLine();
+	var contactsReference = $"contacts/{id}-A";
```
Comment (garayx) [inline src/unit1/lesson7/README.md]: please format the code

File: src/unit1/lesson7/README.md [hunk 3]
```diff
@@ -312,10 +313,11 @@ private void UpdateContact()
...
```
Comment (garayx) [inline src/unit1/lesson7/README.md]: please format the code

**Review by garayx — CHANGES_REQUESTED** [review]

File: src/unit1/lesson7/README.md (hunk 1 above)
Comment (LielNagar) [reply src/unit1/lesson7/README.md]: done

File: src/unit1/lesson7/README.md (hunk 3 above)
Comment (LielNagar) [reply src/unit1/lesson7/README.md]: done

File: src/unit1/lesson7/README.md (hunk 2 above)
Comment (LielNagar) [reply src/unit1/lesson7/README.md]: done

**Review by garayx — APPROVED** [review]

**Pull request merged**